test-environment.yml

```diff
@@ -5,5 +5,7 @@ channels:
   - defaults
 dependencies:
   - bats
+  - grep
   - coreutils
-  - sra-tools  
+  - sra-tools
+  - fastq_utils
```

Comment:
Could you clarify what was dropped in terms of usage that meant that fastq_utils is no longer needed? Or it was never needed?